src/OrchardCore.Modules/OrchardCore.Features/Deployment/AllFeaturesDeploymentSource.cs

```diff
@@ -15,7 +16,9 @@ public AllFeaturesDeploymentSource(IModuleService moduleService)
             _moduleService = moduleService;
         }
 
-        public async Task ProcessDeploymentStepAsync(DeploymentStep step, DeploymentPlanResult result)
+        public override int Order => Int32.MinValue;
+
```

Comment:
LGTM

Couple of comments

For this you could define a very low order as @jtkech suggested.
But not so low that a custom `IDeploymentSource` cannot run before this, a user will want to do this one day ;)

Also can you test, I think, but am not sure, that `ContentDefinitionsStep` also needs to run early (before Contents) or `Contents` can fail.

And lastly, somehow this needs to be reflected in the UI, probably just by sorting based on this Order value, is enough for me. 

Other than that looks like it fixes the issue to me

Reply:
> But not so low that a custom IDeploymentSource cannot run before this, a user will want to do this one day ;)

😄 ok, I will use `-100`

> Also can you test, I think, but am not sure, that ContentDefinitionsStep also needs to run early (before Contents) or Contents can fail.

Ok

> And lastly, somehow this needs to be reflected in the UI, probably just by sorting based on this Order value, is enough for me.

This is good and bad, but let us hear from other folks

Reply:
-1000 ;)